style.css

```diff
@@ -117,3 +117,439 @@ nav .nav-icon {
   padding: 0 0 0 0;
   margin: 0;
 }
+
+.recent-works h1 {
```

Comment:
You have properly laid out the works section. However, it is a requirement for this project for the container of the works section to have a `display:grid` property. Kindly add a rule to handle that.